src/about/hackathon.html

```diff
@@ -0,0 +1,65 @@
+---
+layout: default
+
+title: Hackathon
+description: >
+  We're thrilled to announce the first edition of the DjangoCon Hackathon, happening this year.
+  This is your chance to learn new skills, collaborate with friendly Django developers,
+  and showcase your creativity.
+
+permalink: /hackathon/
+---
+
+<div class="block-container">
+  <div class="wrapper">
+    <header class="flex flex-wrap items-center">
+      <div class="flex-1 max-w-screen-lg">
+        <h1 class="mb-12 pageheading">{{ title }}</h1>
+        <p class="subheading">
+          Welcome to DjangoCon US 2024 Hackathon!
+        </p>
+      </div>
+    </header>
+  </div>
+</div>
+
+<div class="bg-gray-100 block-container">
+  <div class="wrapper">
+    <div class="grid-1-2">
+      <div class="prose lg:prose-lg">
+{% markdown %}
+Are you a beginner eager to dive into Django development? Or a seasoned pro ready to take on new challenges? No matter your skill level, this event is for you!
+
+We're thrilled to announce the first edition of the DjangoCon US Hackathon, happening this year. This is your chance to learn new skills, collaborate with friendly Django developers, and showcase your creativity.
+
+Join us for a day of coding, learning, and fun. Let's build something extraordinary together!
+{% endmarkdown %}
+
+      <a class="button text-gray-500 cursor-not-allowed pointer-events-none" href="#">Registration starts on August 20th!</a>
+      </div>
+
+      <section class="relative flex flex-col self-start p-6 prose bg-white lg:prose-lg bg-lines lg:pt-10 card-effect-3">
+{% markdown %}
+## What to expect:
+
+-   **Registration:** Registration will be available starting August 20, 2024
+-   **Announcement of hackathon participants:** All accepted hackathon participants will receive their acceptance emails by September 3rd, 2024
+-   **Slack group:** All accepted hackathon participants will be added to the hackathon slack group for easier communication
```

Comment:
```suggestion
-   **Slack group:** All accepted hackathon participants will be added to the hackathon Slack channel in the DjangoCon US Attendee Slack workspace for easier communication
```

We are requiring them to have a conference ticket (sprints only is fine), right?

Reply:
@Doreen970 @nzeager See above

Reply:
I think it's reasonable to require them to have a ticket. I could see it being a useful to get people interested in future conferences, so we could consider opening it up if needed. But for simplicity I would stick with only attendees unless anyone else feels that we should do otherwise.

Reply:
If they are only going to attend the hackathon I am sure sprints only will be fine for them.